### PR TITLE
Allow setting the Prometheus instance document title through a flag. 

### PR DESCRIPTION
**Pull request by achiuBAE**
implemented: #4683 
- The HTML title can now be set via a flag: --web.ui-title="[custom name]"

cc @juliusv

### CODE REVIEW

**Comment by juliusv** [comment]
I can actually see this being useful if you have a bunch of tabs with different Prometheus servers open and you want to quickly differentiate them by, say, datacenter name. However, I want to make sure that others think it's a good idea to add this as a flag as well.

@brian-brazil and others?

**Comment by brian-brazil** [comment]
I'm not sure about this one either way to be honest.

**Comment by achiuBAE** [comment]
Hey has there been any decision made or this or does anything else need changing? :)

File: cmd/prometheus/main.go [hunk 1]
```diff
@@ -150,6 +150,9 @@ func main() {
 	a.Flag("web.console.libraries", "Path to the console library directory.").
 		Default("console_libraries").StringVar(&cfg.web.ConsoleLibrariesPath)
 
+	a.Flag("web.ui-title", "Document title of Prometheus instance.").
```
Comment (juliusv) [inline cmd/prometheus/main.go]: I think I'd call this `web.page-title` or `web.html-title`, with preference towards the former.

**Comment by simonpasquier** [comment]
@achiuBAE you need to run `make assets`, commit the changes and push again.

**Comment by simonpasquier** [comment]
LGTM

**Pull request merged**